packages/driver/src/multi-domain/domain_fn.ts

```diff
@@ -39,18 +40,19 @@ export const handleDomainFn = (Cypress: Cypress.Cypress, cy: $Cy, specBridgeComm
     // Set the state ctx to the runnable ctx to ensure they remain in sync
     cy.state('ctx', cy.state('runnable').ctx)
 
-    // Stability is always false when we start as the page will always be
-    // loading at this point
-    cy.isStable(false, 'multi-domain-start')
+    cy.state('isMultiDomain', true)
```

Comment:
You can also tell if you're in 'multi-domain' by checking for `window.specBridgeDomain`. We set that in the injection code so it's present on all multi-domain windows.

Reply:
If not i wonder if you should set this in setup, since it's always true right?

Reply:
The state gets reset here which is why I'm setting it here but I'm not opposed to using a different mechanism either.

Reply:
I logged an issue to the Miro board about this topic so we can discuss it further and decide what's the best way to determine if we are in multi-domain.